javascript/javascript1/week3/saveNote.js

```diff
@@ -0,0 +1,40 @@
+//save a note.
+let notes = [];
+
+function saveNote(content, id) {
+  notes.push({ content: content, id: id });
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do cooking", 2);
+saveNote("Go to work", 3);
+saveNote("Go for a walk", 4);
+
+console.log(notes);
+
+//Get a note
+
+function getNote(id) {
+  for (i = 0; i < notes.length; i++) {
+    if (notes[i].id === id) {
+      return notes[i];
+    } else return "Error";
```

Comment:
instead of this line you can throw error like this
```js
 if (notes[i].id === id) {
      return notes[i];
    }
throw new Error('note not found!');
```